crates/mitex/src/lib.rs

```diff
@@ -476,9 +611,39 @@ static DEFAULT_SPEC: once_cell::sync::Lazy<CommandSpec> = once_cell::sync::Lazy:
 #[cfg(test)]
 mod tests {
     use insta::{assert_debug_snapshot, assert_snapshot};
+    use mitex_parser::spec::CommandSpec;
+
+    static DEFAULT_SPEC: once_cell::sync::Lazy<CommandSpec> = once_cell::sync::Lazy::new(|| {
```

Comment:
Since there is already a `DEFAULT_SPEC` in `crate`, we can import it by `use crate::DEFAULT_SPEC`;